include/seqan3/alphabet/nucleotide/all.hpp

```diff
@@ -105,21 +107,11 @@
  *     // ...
  * ~~~
  *
- * All alphabets define at least the letters `::%A`, `::%C`, `::%G`, `::%T`, `::%U`, `::%UNKNOWN` (although some
- * might be aliases of others).
+ * All alphabets define all letters, although some resolve to the same internal values.
  *
  * \par Printing and conversion to char
  *
- * | to_char()      | seqan3::dna15  | seqan3::dna5 | seqan3::dna4 | seqan3::rna15  | seqan3::rna5 | seqan3::rna4 |
- * |:--------------:|:--------------:|:------------:|:------------:|:--------------:|:------------:|:------------:|
- * | type::A        |    `'A'`       |    `'A'`     |    `'A'`     |    `'A'`       |    `'A'`     |    `'A'`     |
- * |   ...          |    ...         |    ...       |    ...       |    ...         |    ...       |    ...       |
- * | type::T        |    `'T'`       |    `'T'`     |    `'T'`     |    `'U'`       |    `'U'`     |    `'U'`     |
- * | type::U        |    `'T'`       |    `'T'`     |    `'T'`     |    `'U'`       |    `'U'`     |    `'U'`     |
- * | type::R        |    `'R'`       |     ---      |    ---       |    `'R'`       |     ---      |     ---      |
- * |   ...          |    ...         |     ---      |    ---       |    ...         |     ---      |     ---      |
- * | type::N        |    `'N'`       |    `'N'`     |    ---       |    `'N'`       |    `'N'`     |     ---      |
- * | type::UNKNOWN  |    `'N'`       |    `'N'`     |    `'A'`     |    `'N'`       |    `'N'`     |    `'A'`     |
+ * The character values can be printed by `to_char()`.
```

Comment:
`the to_char member function`

Reply:
No, also and especially the free function`to_char`!

Reply:
then `seqan3::to_char` to be explicit?